crates/re_viewport_blueprint/src/container.rs

```diff
@@ -57,7 +57,7 @@ impl ContainerBlueprint {
                 .copied(),
         );
         let Some(container_kind) = results.get_instance::<ContainerKind>(resolver, 0) else {
-            re_log::error!("Container {id:?} is lacking the required `ContainerKind` component.");
+            re_log::error_once!("Container is lacking the required `ContainerKind` component. Maybe you are loading an old blueprint?");
```

Comment:
This shouldn't even be an error-once. This is a totally valid state to be in once a container / space-view has been removed and before the tree has been cleaned up via GC.

Reply:
So we should just remove the log line then>

Reply:
And the equivalent in `re_viewport_blueprint/src/space_view.rs`